.eslintrc.js

```diff
@@ -119,10 +119,8 @@ module.exports = {
             'playground/**/safe-mode-credential-enrollment-intent.json',
             'playground/**/safe-mode-optional-enrollment.json',
             'playground/**/safe-mode-required-enrollment.json',
-            'playground/**/terminal-enduser-email-consent-denied.json',
             'playground/**/terminal-polling-window-expired.json',
             'playground/**/terminal-registration.json',
-            'playground/**/terminal-return-email-consent-denied.json',
             'playground/**/terminal-return-email-consent.json',
             'playground/**/terminal-return-email.json',
```

Comment:
A note - I removed this mock from engleaks spec file here - https://github.com/okta/okta-signin-widget/pull/1892/files

However, since the key `idx.return.to.original.tab` is translated to `oie.return.to.original.tab` in TerminalView - https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/TerminalView.js#L92-L93

I am _not_ removing this line intentionally.

Reply:
Can we just ignore this key https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js#L18 instead of the entire mock file ?